### DISCUSSION TITLE
salut comment bipass le compte administrateur sur Y13 mais de  production: 2022 fin de garantie septembre 2026 .

### DISCUSSION THREAD

**Discussion by DK77550** [community/community#126695] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

salut comment bypass le compte administrateur sur Y13 mais de  production: 2022 fin de garantie septembre 2026 .

**Comment by ebndev** [comment]
Hi @DK77550, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
